handlers/workloads.go

```diff
@@ -0,0 +1,38 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/kiali/kiali/log"
+	"github.com/kiali/kiali/prometheus"
+)
+
+// WorkloadMetrics is the API handler to fetch metrics to be displayed, related to a single workload
+func WorkloadMetrics(w http.ResponseWriter, r *http.Request)  {
+  getWorkloadMetrics(w, r, prometheus.NewClient);
+}
+
+// getWorkloadMetrics (mock-friendly version)
+func getWorkloadMetrics(w http.ResponseWriter, r *http.Request, promClientSupplier func() (*prometheus.Client, error)) {
+  vars := mux.Vars(r)
```

Comment:
(minor) looks like you have some indenting issues? They should go out with `go fmt` afaik